client/angular/pages/profile_page/profile_page_controller.coffee

```diff
@@ -44,6 +44,18 @@ $controller = ($scope, $rootScope) ->
         fragment: 'deactivate_user'
       submit: -> ModalService.open 'ConfirmModal', confirm: confirm
 
+  @deleteUser = ->
+    if AbilityService.canDeactivateUser()
```

Comment:
We'll want to clear this up some and perhaps move it to shared, but I'm happy with this for now. The code I'd want to write:

```
{ deleteConfirmation, deactivateConfirmation } = require 'shared/helpers/user'

@deactivateUser = ->
  ModalService.open 'ConfirmModal', confirm: deactivateConfirmation

@deleteUser = ->
  ModalService.open 'ConfirmModal', confirm: deleteConfirmation
```